File: Backend/Backend_ProyectoFinal/Backend_ProyectoFinal/Models/Actividad.cs
namespace Backend_ProyectoFinal.Models
{
    public class Actividad
    {
        private int idActividad;
        private string nombre;
        private string descripcion;
        private TipoDeActividad tipo;

        public Actividad(int idActividad, string nombre, string descripcion, TipoDeActividad tipo)
        {
            this.IdActividad = idActividad;
            this.Nombre = nombre;
            this.Descripcion = descripcion;
            this.Tipo = tipo;
        }

        public Actividad()
        {
        }

        public int IdActividad { get => idActividad; set => idActividad = value; }
        public string Nombre { get => nombre; set => nombre = value; }
        public string Descripcion { get => descripcion; set => descripcion = value; }
        public TipoDeActividad Tipo { get => tipo; set => tipo = value; }
    }
}

File: main/java/com/example/appmovil/model/Actividad.java
package com.example.appmovil.model;

public class Actividad {
    private int idActividad;
    private String nombre;
    private String descripcion;

    public int getTipo() {
        return tipo;
    }

    public void setTipo(int tipo) {
        this.tipo = tipo;
    }

    private int tipo;

    public Actividad(int idActividad, String nombre, String descripcion, int tipo)
    {
        this.idActividad = idActividad;
        this.nombre = nombre;
        this.descripcion = descripcion;
        this.tipo = tipo;
    }

    public Actividad()
    {
    }

    public int getIdActividad() {
        return idActividad;
    }

    public void setIdActividad(int idActividad) {
        this.idActividad = idActividad;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }


}
